app/celery.py

```
@@ -0,0 +1,38 @@
+"""
+Celery 설정.
+
+Redis를 브로커 및 백엔드로 사용하여 비동기 작업을 처리합니다.
+"""
+
+import os
+from celery import Celery
+from app.core.settings import settings
```

Flake8 (7.3.0)

[error] 'app.core.settings.settings' imported but unused

(F401)